.github/workflows/r-cmd-check.yml

```diff
@@ -51,7 +51,7 @@ jobs:
         with:
           cache-version: 2
           needs: check
-          pak-version: 'devel'
+          pak-version: 'stable'
```

Comment:
`stale` was not working before so we had taken the `devel` version. But now `stable` is working so we can use it.